frontend/components/text-classifier/header/TextClassificationHeader.vue

```diff
@@ -162,6 +164,18 @@ export default {
         records: clearedRecords,
       });
     },
+    async onReset(records) {
+      const resetRecords = records.map((record) => {
+        return {
+          ...record,
+          currentAnnotation: record.annotation,
+        };
+      });
+      this.resetRecords({
+        dataset: this.dataset,
+        records: resetRecords,
```

Comment:
naming => since it's a variable, you mean resetedRecords not the action resetRecord